examples/rosettacode/bitmap_test.nit

```diff
@@ -0,0 +1,43 @@
+# This file tests the Bitmap class in the bitmap module
+# It has to be called by passing two arguments:
+#
+#	- An input bitmap file
+#	- An output bitmap file
+#
+# It converts the input bitmap to grayscale and save it to the specified 
+# output file. In addition, it creates a blank.bmp file in the current directory.
+#
+module bitmap_test
+import bitmap
+
+if args.length < 2 then
+	print "Please specify an input and an output file"
+	return
+end
+var input_path = args[0]
+if not input_path.to_path.exists then
+	print "Input file not found"
+	return
+end
+
+# Loads the bitmap pointed by input_path
+var bitmap = new Bitmap.load(input_path)
+# Converts to grayscale
+bitmap.grayscale
+# Saves the bitmap to a file
+bitmap.save(args[1])
+
+# Create a second bitmap with a width of 400 pixels and a height of 300 pixels.
+# By default, all pixels will have the value of 0xFFFFFF (white)
+var bitmap2 = new Bitmap.with_size(400, 300)
+
+# Change the color of half of the bitmap
+for y in [0..300]
+do
+	for x in [0..200]
+	do
+		bitmap2.set_pixel(x, y, 0x0077AAAA)
+	end 
+end
+# Save it to blank.bmp
+bitmap2.save("./blank.bmp")
```

Comment:
I do not know what kind of text editor you are using but it forgot to add a newline character after the last line